static/src/stylesheets/module/journalism/_campaigns.scss

```diff
@@ -0,0 +1,230 @@
+
+.campaign--snippet {
+    border-top: 1px $garnett-neutral-4 solid;
+    background: $garnett-neutral-8;
+    position: relative;
+    padding: 0 0 6px;
+    margin: 16px 0 36px;
+    font-size: 14px;
+    line-height: 21px;
+
+    &:not([open]) .is-on,
+    &[open] .is-off {
+        display: none;
+    }
+}
+
+/* SUMMARY */
+
+.campaign--snippet > summary {
+    outline: none;
+    padding: 0;
+    border-top: 1px $garnett-neutral-4 solid;
+    list-style: none;
+
+    &::-webkit-details-marker {
+        display: none;
+    }
+
+    .campaign--kicker {
+        display: flex;
+        flex-direction: row;
+
+        > .campaign--snippet__heading-logo {
+            flex: initial;
+            margin-right: 10px;
+
+            .speech-bubble {
+                background-color: $news-garnett-main-1;
+                padding: 6px 10px 10px;
+                color: #ffffff;
+                line-height: 18px;
+            }
+        }
+        > .heading {
+            flex: auto;
+            font-weight: bold;
+        }
+    }
+}
+
+.campaign--snippet__header {
+    margin: 0 0 16px;
+}
+
+.campaign--snippet__headline, .campaign--snippet__heading-logo {
+    font-size: 18px;
+    line-height: 22px;
+    font-family: $f-serif-text;
+    font-weight: 500;
+}
+
+.campaign--snippet__handle {
+    background: #121212;
+    color: #ffffff;
+    font-family: $f-sans-serif-text;
+    font-size: 13px;
+    position: absolute;
+    bottom: 0;
+    transform: translate(0, 50%);
+    padding: 0 15px 0 7px;
+    margin-left: 5px;
+    span {
+        display: inline-flex;
+        align-items: center;
+    }
+    .icon {
+        fill: #ffffff;
+        width: 20px;
+        height: 20px;
+        margin-right: 10px;
+    }
+}
+
+.campaign__button--rounded {
+    border-radius: 100em;
+}
+.campaign__button--large {
+    height: 2.25em;
+}
+
+.campaign__button {
+    display: -webkit-inline-box;
+    display: -ms-inline-flexbox;
+    display: inline-flex;
+    -webkit-box-align: center;
+    align-items: center;
+    border: 0;
+    margin: 0;
+}
+
+.content--pillar-news .campaign--snippet__handle:focus {
+    background-color: $news-garnett-main-1;
+}
+
+svg:not(:root) {
```

Comment:
Hey @ajwl! This had a knock on affect on some of our svgs. See the example on the quote here: 
![image](https://user-images.githubusercontent.com/8774970/39201846-303642d8-47e8-11e8-8f79-d66771541d6f.png)

Does this need to be on all the svgs?

Reply:
😮 I should have seen this, my bad! Will push an update. Thanks for raising 🙌

Reply:
Gosh no problem!